web/api/v1/api.go

```diff
@@ -586,6 +586,7 @@ func respond(w http.ResponseWriter, data interface{}) {
 	w.Header().Set("Content-Type", "application/json")
 	w.WriteHeader(http.StatusOK)
 
+	json := jsoniter.ConfigCompatibleWithStandardLibrary
 	b, err := json.Marshal(&response{
```

Comment:
One issue we had here too was serializing a byte slice and then writing it to `w` rather than directly writing into `w`. With multi-megabyte sized responses, that's probably quite a bit of slowdown. Can we fix this while we are at it.

Reply:
That doesn't appear possible with the APIs.

Reply:
Mh, I just checked the godoc and am a bit puzzled. The `jsoniter.ConfigCompatibleWithStandardLibrary` variable is a [`Config`](https://godoc.org/github.com/json-iterator/go#Config) object. From all I can see this object does not have a `Marshal` method as it is being used here and shouldn't compile. It seems `Config.Froze` must be called to retrieve an object with the Marshal method. Is the vendored dependency possible out of data?

The [`API`](https://godoc.org/github.com/json-iterator/go#API)` the froze method would return then also has a `NewEncoder` method which we can use with the response writer to do it without the extra byte slice.